docker-compose.yml

```diff
@@ -78,7 +78,7 @@ services:
       ${WEBAPP_DOMAIN}: ${WEBAPP_HOST_IP}
       ${KEYCLOAK_DOMAIN}: ${KC_HOST_IP}
     ports:
-      - '80:80'
+      - '81:80'

```

Comment:
Was this is a necessary change?

Reply:
It's not necessary, I just was trying to run the webapp and the ngninx server at the same time and they want the same ports.